### PR TITLE
Remove the IO dispatcher since Room will take care of threading, add …

### PR DESCRIPTION
**Pull request by ceruleanotter**
…a conflict strategy to avoid crash

### CODE REVIEW

File: app/src/main/java/com/example/android/roomwordssample/WordDao.kt [hunk 1]
```diff
@@ -40,10 +41,7 @@ interface WordDao {
     @Query("SELECT * from word_table ORDER BY word ASC")
     fun getAlphabetizedWords(): LiveData<List<Word>>
 
-    // We do not need a conflict strategy, because the word is our primary key, and you cannot
```
Comment (florina-muntenescu) [inline app/src/main/java/com/example/android/roomwordssample/WordDao.kt]: Could you either: 
1. Make this change on the Java branch as well
2. Revert this change in this PR

Comment (ceruleanotter) [reply app/src/main/java/com/example/android/roomwordssample/WordDao.kt]: Done, see https://github.com/googlecodelabs/android-room-with-a-view/pull/18
